blogs/nlp/LOREN/README.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Logic-Regularized Reasoning for Interpretable Fact Verification
+author: Ziyue Wang
+date: 2022-4-8
+tags:
+  - Fact verafication
+  - Logic-regularized neural latent model
+  - LOREN
```

Comment:
could remove this tag

Reply:
I have removed the tag LOREN on my fork, I will make a new commit later when I add the example pictures for Decomposition of the claim and Probing questions generation, which is just illustration example created by pdf editor.